third_party/skcms/version.sha1

```diff
@@ -1 +1 @@
-668026c511f3d4be9447b0ae28ea7a73b5899262
```

Comment:
I don't know if we should include this into the cherry fix as it might cause additional conflicts. This already looks different from the upstream.